moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -529,6 +529,34 @@ bool MoveItConfigData::outputFakeControllersYAML(const std::string& file_path)
     emitter << YAML::EndSeq;
     emitter << YAML::EndMap;
   }
+
+  emitter << YAML::EndSeq;
+
+  // Add an initial pose for each group
+  emitter << YAML::Key << "initial" << YAML::Comment("Default robot poses can be defined here");
+  emitter << YAML::Value << YAML::BeginSeq;
+
+  bool found = false;
```

Comment:
`found` isn't yet used. However, you could use it for a commented-out template as above.

Reply:
Done